assets/global.js

```diff
@@ -543,13 +543,28 @@ class SliderComponent extends HTMLElement {
       this.nextButton.removeAttribute('disabled');
     }
 
+    if (this.currentPage != previousPage) {
+      this.dispatchEvent(new CustomEvent('slideChanged', { detail: {
+        currentPage: this.currentPage
+      }}));
+    }
+
+    if (!this.pageCount || !this.pageTotal) return;
+
     this.pageCount.textContent = this.currentPage;
     this.pageTotal.textContent = this.totalPages;
   }
 
+  scrollIntoView(element) {
+    const lastVisibleSlide = this.slider.clientWidth + this.slider.scrollLeft - 10;
+    const isSlideVisible = element.offsetLeft < lastVisibleSlide && element.offsetLeft > this.slider.scrollLeft;
+    if (!isSlideVisible) this.slider.scrollTo({ left: element.offsetLeft });
+  }
+
   onButtonClick(event) {
     event.preventDefault();
-    const slideScrollPosition = event.currentTarget.name === 'next' ? this.slider.scrollLeft + this.sliderLastItem.clientWidth : this.slider.scrollLeft - this.sliderLastItem.clientWidth;
+    const step = event.currentTarget.dataset.step || 1;
```

Comment:
The steps with also the possibility to drag on mobile can create some situation where you can't click the next/previous button: [video](https://screenshot.click/21-11-flgn8-vmm63.mp4)

Reply:
Seems to only do it when scrolling back. Starting at the end to go back to the beginning.